client/src/components/common/ImgCarousel/ImgCarousel.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import StyledCarousel from './styles';
+
+import forgotPassImage from '../../../assets/img/forgotPassImage.png';
+import LogInImage from '../../../assets/img/LogInImage.png';
```

Comment:
```suggestion
import LogInImage from '../../../assets/img/logInImage.png';
```
change all image files